src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -129,14 +129,16 @@ internal static string ReturnCommonPathRootFromTargetSpecifiersIfOneExists(IEnum
 
         public override int Run(AnalyzeOptions analyzeOptions)
         {
-            if (!Environment.GetCommandLineArgs().Any(arg => arg.Equals("--sarif-output-version")))
+            if (!Environment.GetCommandLineArgs().
+                Any(arg => arg.Equals("--sarif-output-version", StringComparison.OrdinalIgnoreCase) ||
+                arg.Equals("-v", StringComparison.OrdinalIgnoreCase)))
```

Comment:
this is not related to the change,
the code clearly missed the `-v` variant so I added it btw.